mage/svu/mage.go

```diff
@@ -70,3 +79,52 @@ func SVUWithOptions(ctx context.Context, opts ...svudagger.Option) error {
 
 	return nil
 }
+
+// loads environment variables into options.
+func loadEnvVars(opts []svudagger.Option) ([]svudagger.Option, error) {
```

Comment:
No need to pass the original options in - just append them to the returned list in the call above.
```suggestion
func loadEnvVars() ([]svudagger.Option, error) {
```